lib/uri_service.rb

```diff
@@ -7,6 +7,8 @@ module URIService
     'boolean' => '_bi'
   }.freeze
 
+  PER_PAGE_DEFAULT = 20
```

Comment:
I wonder if it's worth making this a config option in uri_service.yml.  If you do, then you could supply that value to the kaminari_config.rb initializer so we don't set `config.default_per_page = 20` and `PER_PAGE_DEFAULT = 20` in two different places.

Reply:
And then if we ever want to increase or decrease the config value, we can do so without having to re-deploy the app.  And as another side effect, we could set different values for the dev/test/prod environments if that's useful for any reason.

Reply:
Do you think increasing the per_page default would require releasing a new version of the api? Because functionally we are changing a core feature of the api.

Reply:
Here's why I don't think so: a client really shouldn't be relying on our defaults if the number of returned results may break their code.  If result count is important, then they should supply the per_page param.  I mostly want to avoid hard-coding the number 20 in two places, so I'm happy with any solution that does that (even if we don't add a config option in uri_service.yml).

Reply:
Interesting, that does make sense. I don't think it should be in the config. But I can figure out a solution where the number 20 isn't stored twice.

Reply:
Great, thanks!